internal/olm/operator/registry/index_image.go

```diff
@@ -408,7 +409,7 @@ func (c IndexImageCatalogCreator) UpdateCatalog(ctx context.Context, cs *v1alpha
 				return fmt.Errorf("specifying the bundle add mode is not supported for File-Based Catalog bundles and index images")
 			}
 
-			err = c.runFBCUpgrade(ctx)
+			err = c.runFBCUpgrade(ctx, skipTLSVerify, useHTTP)
```

Comment:
```suggestion
			err = c.runFBCUpgrade(ctx)
```
The `c` is an `IndexImageCatalogCreator` which already has those as members.

```go
type IndexImageCatalogCreator struct {
    SkipTLS         bool
    SkipTLSVerify   bool
    UseHTTP         bool
```

Reply:
modified.